readme.md

```diff
@@ -44,6 +44,10 @@ Any `.node` files included will also support binary relocation.
             // optional, permit entire filterAssetBase emission
             // eg `const nonAnalyzable = process.cwd()` can emit everything in the cwd()
             emitFilterAssetBaseAll: false,
+            // optional, custom functional asset emitter
+            // takes an asset path and returns the replacement
+            // or returns false to skip emission
+            customEmit: (path, id, isRequire) => false | '"./custom-replacement"',
```

Comment:
I think it could be better to use a single object for params her instead of 3 params

`function customEmit({path, id, isRequire}) { /* body here */ }`

I find this makes it easier in the future if the API changes.

Reply:
Good point, agreed. I usually keep the first arg as the "subject" in these cases so it can be thought of as subject and context, especially when context can be an optional argument for some cases.